components/autofill/src/sync/credit_card/mod.rs

```diff
@@ -78,7 +79,7 @@ pub(crate) struct CreditCardPayload {
 
     // For some historical reason and unlike most other sync records, creditcards
     // are serialized with this explicit 'entry' object.
-    entry: PayloadEntry,
+    pub(crate) entry: PayloadEntry,
```

Comment:
I think these pub changes are only needed because of the visibility of `UnknownEntry` - it's probably not necessary if you change that as I note below.